awx/main/analytics/collectors.py

```diff
@@ -46,7 +46,8 @@ def config(since):
             'release': platform.release(),
             'type': install_type,
         },
-        'system_uuid': settings.SYSTEM_UUID,
+        'install_uuid': settings.INSTALL_UUID,
```

Comment:
FYI @benthomasson I'm planning to modify this a bit.

`install_uuid` is a unique per-AWX-installation UUID
`instance_uuid` is a unique per-server/container UUID